Lib/glyphsets/data.json

```diff
@@ -16432,7 +16432,8 @@
         "GF_Greek_Pro",
         "GF_Latin_Beyond",
         "GF_Latin_Vietnamese",
-        "GF_TransLatin_Arabic"
+        "GF_TransLatin_Arabic",
+        "GF_Latin_PriAfrican"
```

Comment:
Does order here matter?

Reply:
No. These are sets a glyph is in. I guess sorted by alphabetical order would be cleaner.

Anyway, @yanon’s #109 aims to make this file obsolete.